### DISCUSSION TITLE
iOS Missing privacy manifest issues

### DISCUSSION THREAD

**Discussion by apoorvadixit-ttc** [AdamEssenmacher/GoogleApisForiOSComponents#52] [Q&A]
Does this library take care of the **ITMS-91061 Missing privacy manifest** issue that is raised by Apple when submitting the MAUI .net-8 iOS apps? We have been using the Xamarin.Firebase.iOS.Analytics nuget package till now.

**Comment by kim-softnotik** [comment]
This lib works about the "Missing privacy manifest" problem at Apple
